sig-contributor-experience/charter.md

```diff
@@ -71,26 +65,42 @@ We cross-cut all SIGs and WGs to deliver the following processes:
-- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
-- The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- [kubernetes/community] Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
+- User community relations and management.
+  - Slack or YouTube moderation (This should be a new SIG or the CNCF)
+- The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Project Organizational Management] subproject README.
+- Steering committee election policy updates and maintenance. (Steering Comittee handles this)
 - We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
-- Communication platforms that are out of our scope for maintenance and support but we may still have some influence:
-    - [r/kubernetes]
-    - [@kubernetesio] twitter account
-    - [kubernetes blog]
+- Ongoing maintenance of established SIG Resources. We are available to assist SIGs with these properties and provide guidance, but the responsibility of managing these day to day belongs to the individual SIG.
+  - SIG Calendars and their settings
+  - SIG Mailing List permissions, moderation, and maintenance
+  - SIG YouTube playlist videos
+  - SIG settings on their Zoom account
+- Ownership, staffing, and moderation of Kubernetes communication properties. The CNCF may delegate admin or management positions on these properties to members of Contributor Experience:
+  - [discuss.kubernetes.io] - This tool is mostly self-moderating and enough Kubernetes members participate that most of these roles are administrative.
+  - [Mailing lists] / Google groups (this belongs to the each SIG)
+  - [Slack] (CNCF)
+  - [/kubernetescommunity] YouTube channel (CNCF)
+  - [Zoom] (CNCF)
+  - Ownership of the Kubernetes Gsuite (this is Steering Committee)
```

Comment:
Doesn't it conflict with https://github.com/kubernetes/community/pull/3526/files#diff-ecc1c053d46fb60e012bf855c9630608R29?

Also, 'G Suite' (not 'Gsuite') :)

Reply:
One is ownership of the suite, which belongs to SC, the other is usage of the suite, which is contribex.